### PR TITLE
adding fix for labels pushdown on workflow job nodes

### PR DESCRIPTION
**Pull request by sarabrajsingh**
fixes labels and instance_group ManyToMany IDs not being pushed down to nodes that are spawned from parent templates.

### CODE REVIEW

**Review by AlanCoding — APPROVED** [review]

File: awx/main/models/workflow.py [hunk 1]
```diff
@@ -114,6 +114,9 @@ def _get_workflow_job_field_names(cls):
             'credentials',
             'char_prompts',
             'all_parents_must_converge',
+            'labels',
```
Comment (john-westcott-iv) [inline awx/main/models/workflow.py:117]: Are there any other objects in here that we need add fields to? WorkflowJobTemplateNode?

Comment (john-westcott-iv) [reply awx/main/models/workflow.py:117]: We also have a failing test.

Comment (sarabrajsingh) [reply awx/main/models/workflow.py:117]: test is fixed.

Comment (sarabrajsingh) [reply awx/main/models/workflow.py:117]: > Are there any other objects in here that we need add fields to? WorkflowJobTemplateNode?

we need to think of any object in workflow.py that essentially executes a "copy". i will take a look.

**Pull request merged**